assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -111,9 +137,20 @@ class Draggable extends Component {
 		this.cloneWrapper.style.transform = clone.style.transform;
 
 		// Position clone over the original element.
-		this.cloneWrapper.style.top = `${ getPixelsFromPercentage( 'y', parseInt( clone.style.top ) ) }px`;
-		// Add 5px adjustment for having the block mover right next to the clone.
-		this.cloneWrapper.style.left = `${ getPixelsFromPercentage( 'x', parseInt( clone.style.left ) ) }px`;
+		if ( 'amp/amp-story-text' === blockName ) {
+			// Add an adjustment for having the block mover right next to the clone.
+			const pattern = /calc\(([\d\.]+)%/;
+			const matchesTop = clone.style.top.match( pattern );
+			const percentageTop = matchesTop ? matchesTop[ 1 ] : 0;
```

Comment:
For example, if `clone.style.top` was `calc(98.53%)`, `percentageTop`, would be `98.53`